src/main/resources/org/biouno/unochoice/stapler/unochoice/unochoice.js

```diff
@@ -688,45 +497,14 @@ var UnoChoice = UnoChoice || ($ => {
                 this.originalArray.push(options[i]);
             }
         } else if (paramElement.tagName === 'DIV' || paramElement.tagName === 'SPAN') { // handle CHECKBOXES
-            if ($(paramElement).children().length > 0 && paramElement.children[0].tagName === 'TABLE') {
-                let table = paramElement.children[0];
-                let tbody = table.children[0];
-                if (paramElement.className === 'dynamic_checkbox') {
-                    let trs = $(tbody).find('tr');
-                    for (let i = 0; i < trs.length ; ++i) {
-                        let tds = $(trs[i]).find('td');
-                        let inputs = $(tds[0]).find('input');
-                        let input = inputs[0];
-                        this.originalArray.push(input);
-                    }
-                } else {
-                    let trs = $(tbody).find('tr');
-                    for (let i = 0; i < trs.length ; ++i) {
-                        let tds = $(trs[i]).find('td');
-                        let inputs = $(tds[0]).find('input');
-                        let input = inputs[0];
-                        this.originalArray.push(input);
-                    }
-                }
-            } // if ($(paramElement).children().length > 0 && paramElement.children[0].tagName === 'TABLE') {
             if ($(paramElement).children().length > 0 && (paramElement.children[0].tagName === 'DIV' || paramElement.children[0].tagName === 'SPAN')) {
                 let tbody = paramElement.children[0];
-                if (paramElement.className === 'dynamic_checkbox') {
```

Comment:
The code in the `if` block and `else` block were identical. I've removed the conditional logic.